### ISSUE TITLE
Some recipes still don't work after #666

### ISSUE THREAD

**Issue by SpeeeDCraft** [GregTechCEu/GregTech#1167]
### GregTech CEu Version

v2.4.1-beta

### Modpack Used

TerraFirmaGreg-R4.1.1-Normal

### Addons Installed

GTFO, GCYM, HtmlTech, DevTech

### Environment

Everywhere

### New Worlds

No

### Cross-Mod Interaction

Unsure

### Expected Behavior

A recipe with a block of clay should work, but it doesn't work, but it doesn't do it.

### Actual Behavior

The recipe doesn't work.

### Steps to Reproduce

Put a block of clay in the macerator.

### Additional Information

JEI:
![image](https://user-images.githubusercontent.com/52341158/187569689-c9289eb9-7a3c-4df3-ab60-afbe11953904.png)
World (There is no result):
![image](https://user-images.githubusercontent.com/52341158/187569728-32d97278-85bb-4d28-a803-9168b8e077c3.png)

I am almost 100% sure that this is because of TFC, because half of the recipes in the modpack have stopped working again. I've already written about this error with this before, and some of the recipes really worked. But there are still some recipes that don't work after #666.

**Comment by SpeeeDCraft** [comment]
I found out a little more information, maybe this will somehow help fix it, recipes stop working, in the event that the input item has recipes in the campfire (HeatingRecipes) in TFC, if you delete this recipe via CT, then the recipe magically starts working fine in GT machines.

**Comment by SpeeeDCraft** [comment]
After a bunch of tests that I did, I found out that the problem is due to this. My tests also confirmed this. If you change the code so that the registration of capability for items occurs after adding recipes, then everything works as intended, this is not a GTCEu problem, so I will close it, I will also fix it myself in the TFC fork.

https://github.com/TerraFirmaCraft/TerraFirmaCraft/blob/23e6a438e5627da1203e0b199e1556db8ed5234f/src/main/java/net/dries007/tfc/api/capability/size/CapabilityItemSize.java#L39
